README.md

```diff
@@ -1,11 +1,270 @@
+````
+
+### Script instantiation / applying arguments
+
+Assuming an Agora system already exists, the handling of the new Proxy and Voting
+effects is not different from other Agora effects.
+
+Consult the Haddock documentation of the [Proxy effect](https://mlabs-haskell.github.io/agora-drep/haddock/Agora-Proxy.html) and [Voting effect](https://mlabs-haskell.github.io/agora-drep/haddock/Agora-Effect-Voting.html)
+to see the function signature of both effects.
+
+Specifically, both effects (like all Agora effects) are parameterized by
+the `CurrencySymbol` of the enabling GAT, which will depend on the specific
+Agora system in use. Therefore, the scripts **must** have this argument applied
+before being used.
+
+### Creating an effect UTxO
+
+For effect creation, consult the definition of the datum types of the scripts:
+
+* [ProxyDatum](https://mlabs-haskell.github.io/agora-drep/haddock/Agora-Proxy.html#t:ProxyDatum)
+* [VotingDatum](https://mlabs-haskell.github.io/agora-drep/haddock/Agora-Effect-Voting.html#t:VotingDatum)
+
+For creating a voting effect, the following requirements must be met:
+
+* A UTxO **must** be created at the address of the desired effect.
+* The UTxO **must** contain a valid datum according the data definitions linked previously
+  (i.e: the datum must be of type `VotingDatum`).
+* The UTxO **must** contain.
+
+Any failure to comply with these will make the UTxO unspendable.
+
+### Executing the effect
+
+For a detailed guide on how effects are executed on Agora based systems, consult
+the following Notion documentation:
+
+* [Effects](https://liqwid.notion.site/Effects-3364731305544dc1972221b6d1353a69)
+* [Authority Tokens](https://liqwid.notion.site/Authority-Tokens-b25d2011c8114e04ac9e73514e6b9421)
+
+The voting script cannot be executed directly by a typical Agora system due to
+PlutusV2/PlutusV3 incompatibilities. This means a more involved process is required
+for creation and execution of the effect.
+
+In a previous [section](#the-intended-use-of-the-scripts), this process is sketched.
+
+Again, for more information on this, consult the technical specification, where
+specific interactions between the Voting, Proxy and Minting scripts are thoroughly
+documented.
+
+However, for the purposes of implementing effect creation and execution, any already
+exiting Agora based system should be able to support the new voting effect:
+
+* The voting effect creation works exactly the same (just the datum and effect address
+  are different)
+* The voting effect execution works similarly, but now two executions are required: one
+  for the proxy effect, and another for the proper voting effect.
+
+## Examples
+
+> Use case: an Agora DAO delegates the ADA contained in its treasury to an automated DRep,
+> which can subsequently vote on Cardano governance proposals **when and only when** the
+> DAO members wish it to do so.
+
+This use case is already implemented in [Clarity](www.clarity.vote), but let's see
```

Comment:
`[Clarity](www.clarity.vote)` is acting like a relative link (broken). Should be `[Clarity](https://www.clarity.vote)`